### DISCUSSION TITLE
How to uninstall the svcg package?

### DISCUSSION THREAD

**Discussion by Zeemyo** [voicepaw/so-vits-svc-fork#244] [Q&A] [answered]
I just want to use remove all the packages that related with this program

**Comment by 34j** [comment, marked as answer]
You can just remove the venv.

**Comment by foxiergo** [comment]
Hey, I had 20 GB of free disk space. 
After I ran istall.bat, the boot went naturally (it said 2.6gb). But when the whole load went through, I suddenly had 12 gb of memory down. After using it, I decided to delete everything I wanted to from this program (well uninstall the program to do). I found a folder that weighs 5 gb, and deleted it, but even so I have only 17 gb of space, somewhere else is 3 gb. And where?
